MediaPlayer/MediaPlayer.module.less

```diff
@@ -0,0 +1,7 @@
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
```

Comment:
Is this for future usage?

Reply:
Yes, but I better delete them for now and use them when needed.